shared/types/payment.ts

```diff
@@ -17,6 +21,8 @@ export type Payment = {
   payoutId: string
   payoutDate: Date
   created: DateString
+  stripeTransactionFee: number
```

Comment:
should we just call this `transactionFee`? this would avoid explicitly referring to stripe as much as possible in code

Reply:
yup good point! I put it as `stripeTransactionFee` initially as there were many other types of fees e.g. 
<img width="555" alt="image" src="https://user-images.githubusercontent.com/56983748/227120171-72261850-d061-4510-8962-08657181da95.png"> ([source](https://stripe.com/docs/api/balance_transactions/object)) and I wanted to be clear that this was the fee that Stripe charges. but on on further thought you're right, we can leave it as `transactionFee` since we know what we're referring to

Reply:
[073bdd5](https://github.com/opengovsg/FormSG/pull/5942/commits/073bdd55fcd76bf9fd8d4008cec21e733fee47d4)